### ISSUE TITLE
Hyperlink not applying in right place

### ISSUE THREAD

**Issue by danirzv** [QuestPDF/QuestPDF#747]
**Describe the bug**
In some scenarios Hyperlink not applying on elements and link area is misplaced

**To Reproduce**
```
Document.Create(container =>
{
    container.Page(page =>
    {
        page.ContentFromRightToLeft();
        page.Content().Column(col =>
        {
            col.Item().Hyperlink("https://www.questpdf.com").Text("Link1");
            col.Item().AlignRight().Hyperlink("https://www.questpdf.com").Text("Link2");
            col.Item().AlignCenter().Hyperlink("https://www.questpdf.com").Text("Link3");
        });
    });
}).GeneratePdf("test.pdf");
```
**Expected behavior**
while page is `ContentFromRightToLeft` Hyperlink get placed in right too

**Screenshots**
![image](https://github.com/QuestPDF/QuestPDF/assets/14872370/601f688d-5b57-44cb-a7d1-d0b6f2c4d254)

**Comment by blogcraft** [comment]
You set ContentFromRightToLeft, so the link is starting in the Right as expected.

**Comment by danirzv** [comment]
> You set ContentFromRightToLeft, so the link is starting in the Right as expected.

Exactly, but text is right with no hyperlink, and some area in same line in left has link on with no text! 😅

**Comment by MarcinZiabek** [comment]
@danirzv Thank you for providing a straightforward example.

This issue should be fixed in the latest `2023.12.2` release. I have also backported it to still supported pure-MIT `2022.12.13` release.

Would you please test it? 😀

**Comment by danirzv** [comment]
Thank you very much, yea it works fine now.
Sorry if it took too long to reply, I forgot to respond here! 😅
